modules/react/collection/lib/useOverflowListModel.tsx

```diff
@@ -103,8 +103,10 @@ export const useOverflowListModel = createModelHook({
   const state = {
     ...model.state,
     hiddenIds: internalHiddenIds,
-    itemWidthCache,
-    containerWidth,
+    itemSizeCache,
+    itemWidthCache: itemSizeCache,
+    containerSize,
+    containerWidth: containerSize,
```

Comment:
Add deprecated message here:

```suggestion
    /**
    * @deprecated Use `containerSize` instead
    */
    containerWidth: containerSize,
```